src/main/java/com/google/edith/servlets/Item.java

```diff
@@ -48,4 +48,18 @@ public static Builder builder() {
 
     public abstract Item build();
   }
+
+  @Override
+  public boolean equals(Object item) {
+    Item itemObject = (Item) item;
+    if (itemObject.name().equals(this.name()) && itemObject.userId().equals(this.userId())) {
+      return true;
+    }
+    return false;
+  }
+
+  @Override
+  public int hashCode() {
+    return this.userId().hashCode() * 11;
```

Comment:
Just verifying we have tests for these right?

Reply:
just submitted tests